File: roles/dev/files/java/src/test/java/com/example/util/UtilTests.java
import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.CsvSource;

import com.example.util.Util;

class UtilTests {
    @ParameterizedTest
    @CsvSource({
        "One, Two"
    })
    void joinStrings_concatenates_with_comma_and_space(
        final String a,
        final String b
    ) {
        assertEquals(String.format("%s, %s", a, b), Util.joinStrings(a, b));
    }
}

File: roles/ja-apex-dev/jf-dotnet/files/dotnet/tests/Template.Tests/Utilities/UtilTests.cs
namespace Template.Tests;

using Template.Utilities;

public class UtilTests
{
    [TestCase("One", "Two")]
    public void JoinStrings_Concatenates_With_Comma_And_Space(string a, string b)
    {
        Assert.That(Util.JoinStrings(new[] { a, b }), Is.EqualTo($"{a}, {b}"));
    }
}
